src/components/shoppingListGrid.tsx

```diff
@@ -10,6 +10,8 @@ import {
   Grid,
 } from '@material-ui/core';
 
+import SingleShoppingList from './singleShoppingList';
```

Comment:
This component file and component can just be called `shoppingList.tsx` and `ShoppingList` respectively. It's a pretty common convention for the singular thing to just be called `thing` (for example) and the plural thing to be called `things`

Reply:
completed